serialize/src/flags.rs

```diff
@@ -91,33 +113,21 @@ impl Flags for SWFlags {
     }
 
     #[inline]
-    fn from_u8(value: u8) -> Self {
+    fn from_u8(value: u8) -> Option<Self> {
         let x_sign = (value >> 7) & 1 == 1;
         let is_infinity = (value >> 6) & 1 == 1;
         match (x_sign, is_infinity) {
-            (_, true) => SWFlags::Infinity,
-            (true, false) => SWFlags::PositiveY,
-            (false, false) => SWFlags::NegativeY,
+            (true, true) => None,
```

Comment:
Why is `true, true` invalid? Is this just to avoid there being two ways to serialize the point at infinity? If so, can there be a comment here to make it explicit that that is why?

Reply:
```suggestion
			// This is invalid because we only want *one* way to serialize
			// the point at infinity.
            (true, true) => None,
```

Reply:
LGTM